src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java

```diff
@@ -70,6 +112,14 @@
  *   &lt;property name=&quot;allowLineBreaks&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
```

Comment:
please add to tokens `METHOD_REF`

and please make config+example of DOT+METHOD_REF  without allowLineBreaks , to show how it become better with this property for such tokens. Example should show multiline method and chain calls.

Reply:
Done

Reply:
this is not done.

Reply:
kin PR description i see example .. please recheck that you pushed code to Github.

Please remove example for DOT only. DOT+METHOD_REF is enough.

Reply:
> kin PR description i see example .. please recheck that you pushed code to Github

The full code was pushed to github with all four configuration

> Please remove example for DOT only. DOT+METHOD_REF is enough.

There is no seperate example-configuration for DOT only. Are you saying to remove DOT+allowLineBreak.

Reply:
We do not need example of:
```
        <module name="NoWhitespaceBefore">
            <property name="tokens" value="DOT"/>
            <property name="allowLineBreaks" value="true"/>
        </module>
```

we need :
```
        <module name="NoWhitespaceBefore">           
             <property name="tokens" value="DOT"/> 
             <property name="tokens" value="METHOD_REF"/> 
        </module>
```
and 
```
        <module name="NoWhitespaceBefore">           
             <property name="tokens" value="DOT"/> 
             <property name="tokens" value="METHOD_REF"/> 
             < property name="allowLineBreaks" value="true"/>
        </module>
```